### ISSUE TITLE
Broken page layout in German

### ISSUE THREAD

**Issue by pyrog** [okfn/opendataday#297]
Hi,

The layout is broken in German DE and DE-AT:

<img width="50%" alt="DE" src="https://user-images.githubusercontent.com/2158081/106309870-029c1580-6263-11eb-8db3-034b1f5ce37e.png">  <img width="50%" alt="DE-AT"  src="https://user-images.githubusercontent.com/2158081/106310390-c74e1680-6263-11eb-9ec1-ea6c64125d23.png">

<img width="50%" alt="English" src="https://user-images.githubusercontent.com/2158081/105964271-3ec45000-6082-11eb-8438-62702de17a33.png">

Also, the translation don't seem up to date 🤔

**Comment by loleg** [comment]
Need help?

**Comment by StephenAbbott** [comment]
@loleg If you would be able to submit a shorter translation of the text here, that would be greatly appreciated

**Comment by loleg** [comment]
I've looked at the broken layout issues in several of the languages. The optimal length of text changes between screen-size breakpoints, and I would suggest that we consider adding a vertical overflow rule to the left-hand paragraph:

```css
#what .about {
    overflow-y: auto;
    overflow-x: hidden;
}
```

Which would look like this:

![odd1](https://user-images.githubusercontent.com/31819/107990262-55502e00-6fd4-11eb-870f-b1a29b08dbde.jpg)

Not quite optimal from a usability point of view, but again the text fits fine on my full-screen browser or mobile view, and I think it's a better solution than trying to hammer down the content across several languages.

An alternative design fix would be to remove the fixed height in this part of the page, letting both left and right columns take up as much space as they need.

Speaking of the right-hand side: here the design blocks us from any scrolling. It is problematic that in German you have three lines of text for the thematic areas. In my screenshot I've greatly reduced the amount of text, and will submit a PR with this suggestion.

**Comment by loleg** [comment]
One more thing a native German speaker commented on is the capitalization of the very top text of the page ("You Are Invited!" / "Du Bist Eingeladen!"), which is problematic in languages like this one which take offense at false capitalization. It would be best if this was controlled by the translator, however this line in [_home.scss](https://github.com/okfn/opendataday/blob/master/assets/scss/_home.scss#L21) forces it in every language:

```
& > a { ... h1 {
 text-transform: capitalize;
```

**Comment by pyrog** [comment]
> One more thing a native German speaker commented on is the capitalization of the very top text of the page 

👍 This is problematic too in french 😉

**Comment by chris48s** [comment]
Hi. Thanks for the feedback on these. I've just pushed a couple of changes.

Firstly I've removed the `text-transform: capitalize;` so this can now be handled in an appropriately language-sensitive way by translators.

Secondly I've increased the height of this so it is tall enough to accommodate the longest text we've got (currently Russian):

![Screenshot at 2021-02-16 15-29-52](https://user-images.githubusercontent.com/6025893/108086562-6f3e4f00-706e-11eb-9f17-8eafd9a1a7eb.png)

this leaves us with a lot of vertical space for terser languages though.

![Screenshot at 2021-02-16 15-29-27](https://user-images.githubusercontent.com/6025893/108086589-75ccc680-706e-11eb-8465-123cba9efa05.png)

Tbh I'm not 100% convinced this fixed-height/diagonal layout is buying us that much. Given one of the characteristics of the site is that it needs to accomodate text of dramtically varying heights here maybe it would be better to drop this design and use a layout which responds better to the size of text e.g:

![Screenshot at 2021-02-16 15-32-17](https://user-images.githubusercontent.com/6025893/108086642-81b88880-706e-11eb-80ea-1a1a21c2b41b.png)

Lets quickly catch up on this next time we speak @StephenAbbott
